prereise/gather/data/BA_County_map.json

```diff
@@ -2830,15 +2830,17 @@
         "Fall River__SD",
         "Lawrence__SD",
         "Meade__SD",
-        "Pennington__SD"
+        "Pennington__SD",
+        "Oglala Lakota__SD"
     ],
     "EPE": [
         "Dona Ana__NM",
         "Otero__NM",
         "Eddy__NM",
         "El Paso__TX",
         "Hudspeth__TX",
-        "Culberson__TX"
+        "Culberson__TX",
+        "Doña Ana__NM"
```

Comment:
Since we're adding `"Doña Ana__NM"`, should we remove `"Dona Ana__NM"`?

Reply:
Yep, will remove.